pages/implementation/inspector/sdk/java.mdx

```diff
@@ -0,0 +1,165 @@
+---
+layout: 'guide'
+title: 'Avo Inspector Java SDK'
+---
+
+import Link from '../../../../components/Link';
+
+## <a name="quick-start"></a>Quick Start Guide
+
+Find the Quick Start Guide in our <a target="_blank" rel="noopener noreferrer" href="https://github.com/avohq/java-avo-inspector">GitHub repo.</a>
+
+## <a name="installation"></a>Installation
+
+We host the library on JitPack.io, so
+
+add the following to the root build.gradle:
+
+```groovy
+repositories {
+    ...
+    maven { url 'https://jitpack.io' }
+}
+```
+
+and:
+
+```groovy
+dependencies {
+    debugImplementation 'com.github.avohq.java-avo-inspector:TAG'
+}
+```
+
+Use the latest GitHub release tag to get the latest version of the library.
+
+## <a name="import"></a>Import
+
+```java
+import is.avo.inspector.AvoEventSchemaType;
+import is.avo.inspector.AvoInspector;
+import is.avo.inspector.AvoInspectorEnv;
+```
+
+## <a name="initialization"></a>Initialization
+
+Obtain the API key in the Inspector tab in your <a target="_blank" rel="noopener noreferrer" href="https://www.avo.app/welcome">Avo.app</a> workspace.
+
+You will need to create an instance of `AvoInspector` with the constructor.
+
+```java
+public AvoInspector(@NotNull String apiKey, @NotNull String appVersion, @NotNull String appName, @NotNull AvoInspectorEnv env)
+```
+
+> #### Parameters:
+>
+> - `String apiKey` - the API key you get in the Inspector tab of your Avo workspace
+> - `String appVersion` - app version to attribute the events to
+> - `String appName` - application name
+> - `AvoInspectorEnv env` - current environment: AvoInspectorEnv.Dev, AvoInspectorEnv.Staging or AvoInspectorEnv.Prod
+
+## <a name="event-tracking"></a>Sending event schemas to Avo Inspector
+
+This is the core of the **Avo Inspector SDK**.
+
+Call \***\*one of the methods\*\*** in this section every time an event is tracked.
+
+### Option 1
```

Comment:
What option do we recommend? Can we set up a "analytics wrapper" example with our recommended approach? 

```public void trackEvent(name, properties)
{
   trackingService.trackEvent(eventName, properties);
   Inspector.trackschema(eventName, properties);
}

Dno, something in that direction?

Reply:
First option covers most of the use cases, like the wrapper function

Reply:
yep, Im thinking of an example like this for them to relate to, so they don't have to think 
"How do i set this up into my code?"

Using option1, if we can create a example wrapper function so for 99% of java users coming in, they can just copy paste without reading too much on the docs and it just works

Reply:
Right, I was not including the examples in the reference docs on purpose, but it probably makes sense to include

Reply:
Added a wrapper example